crates/rooch/src/commands/statedb/commands/inscription.rs

```diff
@@ -136,6 +143,9 @@ pub(crate) fn create_genesis_inscription_store_object(
     let inscription_store = InscriptionStore {
         cursed_inscription_count,
         blessed_inscription_count,
+        //TODO set unbound_inscription_count and lost_sats
+        unbound_inscription_count: 0,
+        lost_sats: 0,
```

Comment:
For statedb/genesis: count lost_sats by charm flag

Reply:
The ordinals has a statistic_to_count db to get the counts [UpgradeCap](https://github.com/ordinals/ord/blob/75bf04b22107155f8f8ab6c77f6eefa8117d9ace/src/index/updater.rs#L604-L626)